Gemfile

```diff
@@ -33,6 +33,11 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'slim'
+gem 'devise'
```

Comment:
A good practice is to add a comment on the line before each new gem include so we know what it does. While devise, slim and omniauth are well-known, devise_token_auth isn't (this is even more important if you add gems on your own without someone mentioning them in the card)